src/core/core.scale.js

```diff
@@ -1255,16 +1259,20 @@ class Scale extends Element {
 			ctx.fillStyle = tickFont.color;
 			ctx.textBaseline = 'middle';
 			ctx.textAlign = item.textAlign;
+			ctx.strokeStyle = optionTicks.strokeStyle;
+			ctx.lineWidth = optionTicks.lineWidth;
```

Comment:
If my memory serves, `ctx` will not accept a `lineWidth` of `0` and remains in the previous value.
So I think we should add a check for `lineWidth` and not do the `strokeText` if its not defined.

Reply:
good one, I did a small test and `ctx` does not accept zeros in `lineWidth` at all. I added a small conditional. thanks!